build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts

```diff
@@ -71,7 +71,12 @@ const getCodeCoverageSummary = (
 	codeCoverageComparisonReport: CodeCoverageComparison[],
 ): string => {
 	const summary = codeCoverageComparisonReport
-		.sort((report1, report2) => report1.branchCoverageDiff - report2.branchCoverageDiff)
+		.sort(
+			(report1, report2) =>
+				report1.branchCoverageDiff +
+				report1.lineCoverageDiff -
+				(report2.branchCoverageDiff + report2.lineCoverageDiff),
+		)
```

Comment:
```suggestion
		.sort(
			(report1, report2) =>
				(report1.branchCoverageDiff + report1.lineCoverageDiff) -
				(report2.branchCoverageDiff + report2.lineCoverageDiff),
		)
```

Is that what you intended? Either way, worth a comment explaining how this sorts the way you want and why.

Reply:
Both mean the same but auto correct changes it to previous one even if I put it in paranthesis.

Reply:
Not worthy of a comment imo. It is very much self explanatory imo.

Reply:
Well... I don't understand what it's doing and why, so can you share the obvious explanation?